Command/RemoveThumbsCommand.php

```diff
@@ -236,4 +257,27 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * @return QuestionHelper|DialogHelper
+     */
+    private function getQuestionHelper()
+    {
+        // NEXT_MAJOR: Remove this BC code for SensioGeneratorBundle 2.3/2.4 after dropping support for Symfony 2.3
+        if (class_exists('Sensio\Bundle\GeneratorBundle\Command\Helper\DialogHelper')) {
+            $questionHelper = $this->getHelper('dialog');
+            if (!$questionHelper instanceof DialogHelper) {
```

Comment:
This can be merge to this `if` statement: https://github.com/sonata-project/SonataMediaBundle/pull/1259/files#diff-b434675e69b14ff482819901a0366360R267